### DISCUSSION TITLE
TimePicker Interval

### DISCUSSION THREAD

**Discussion by justcivah** [dotnet/maui#8646] [Ideas]
Would it be possible to add to the TimePicker a "HourInterval" and a "MinuteInterval" property? This would allow the user to select the time only at the specified interval. For example a TimePicker with MinuteInterval = 30 would allow the user to select **:00 - **:30. A TimePicker with HourInterval = 2 and MinuteInterval = 15 allows 00:00, 00:15, 00:30, 00:45, 02:00, 02:15, 02:30, 02:45, 04:00, etc...

**Comment by johannes-steurer** [comment]
very good idea. Would be most helpful

**Comment by MichaelShapiro** [comment]
very useful and practical idea.

**Comment by ThysBrits** [comment]
Also need to have this on TimePicker, we need to have 15 min intervals.
